src/crawlee/_types.py

```diff
@@ -44,6 +44,8 @@
 
 HttpPayload: TypeAlias = bytes
 
+RequestTransformAction: TypeAlias = Literal['skip', 'unchanged']
```

Comment:
not sure whether this is necessary :slightly_smiling_face:

Reply:
You mean the `TypeAlias` annotation? No, not necessary, but it's nice for readability I suppose.

Reply:
Yeah, why not